code/game/objects/items/crayons.dm

```diff
@@ -380,6 +380,9 @@
 	if(edible && (M == user))
 		user << "You take a bite of the [src.name]. Delicious!"
 		var/eaten = use_charges(5)
+		if(!reagents.total_volume) //Prevents divsion by zero
+			check_empty()
```

Comment:
I added the argument as user a while ago, but I did it from my phone. I just now decided to compile and run it but there's an issue with putting user as an argument. When you finish eating the crayon it outputs "There is no more of green crayon left!" twice instead of just once.
![capture](https://cloud.githubusercontent.com/assets/7967050/18978962/0058da00-8695-11e6-8d99-0d2355b5a9b7.JPG)